src/main/java/competition/subsystems/arm/UnifiedArmSubsystem.java

```diff
@@ -254,6 +260,12 @@ public void setPower(XYPair power) {
     }
 
     public void setArmsToAngles(Rotation2d lowerArmAngle, Rotation2d upperArmAngle) {
+        if (areGivenTargetAnglesIllegal(new XYPair(lowerArmAngle.getDegrees(), upperArmAngle.getDegrees()))) {
+            lowerArm.setPower(0);
```

Comment:
I'm wondering if this gets us in a state where the brake is open and the arm just falls down if we are given illegal angles. If this is a large motion, the maintainer will notice the large error and disable the brake, and if we set zero power, gravity will do its thing.

Reply:
I've completely moved to a different approach in the latest version.